CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKRenderer.java

```diff
@@ -118,6 +118,20 @@ public void AddRenderable(CVKRenderable renderable) {
         }   
     }
     
+    
+    /**
+     * Called from the parent CVKVisualProcessor when the VisualManager is
+     * being destroyed.
+     */
+    public void Destroy() {
+        // TODO HYDRA: Not sure if this is the best method, can it cause a
+        // deadlock? Would it be better to have a shutdown flag and do in the
+        // update loop?
```

Comment:
Not sure we can?  I haven't looked into it but would assume the TopComponent kills us.  It owns the canvas so we can't hang around if it deletes it from under us.  Take a look?

Reply:
Removed comment